src/orquestra/sdk/_base/cli/_dorq/_services/_down.py

```diff
@@ -32,12 +32,11 @@ def __init__(
 
     def on_cmd_call(
         self,
-        manage_ray: Optional[bool],
-        manage_fluent: Optional[bool],
-        manage_all: Optional[bool],
+        ray: Optional[bool],
+        all: Optional[bool],
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏this shadows a global builtin

Reply:
Yeah... I edited the change.